boards/arm/nucleo_f091rc/nucleo_f091rc_defconfig

```diff
@@ -31,6 +31,20 @@ CONFIG_GPIO=y
 CONFIG_GPIO_PINCTRL=y
 CONFIG_GPIO_STM32=y
 
+# SPI Controller 
+# SPI1 - master
+# SPI2 - slave
+CONFIG_SPI=y
+CONFIG_SPI_1=y
```

Comment:
Move to Kconfig.defconfig

Reply:
Ok. will move CONFIG_SPI. CONFIG_SPI_1 will come from DT.